vagrant/install_from_src.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
Done. Exposed a couple of non-obvious bugs too.  Openssl changed their symlinking/hashing scheme!